framework/db/pgsql/Schema.php

```diff
@@ -441,11 +441,11 @@ protected function findColumns($table)
                     $column->defaultValue = bindec(trim($column->defaultValue, 'B\''));
                 } elseif (preg_match("/^'(.*?)'::/", $column->defaultValue, $matches)) {
                     $column->defaultValue = $matches[1];
-                } elseif (preg_match('/^(?:\()?(.*?)(?(1)\))(?:::.+)?$/', $column->defaultValue, $matches)) {
-                    if ($matches[1] === 'NULL') {
+                } elseif (preg_match('/^(\()?(.*?)(?(1)\))(?:::.+)?$/', $column->defaultValue, $matches)) {
```

Comment:
There was a mistake in the regular expression pattern.

http://php.net/manual/en/regexp.reference.subpatterns.php
> If an opening parenthesis is followed by "?:", the subpattern does not do any capturing, and is not counted when computing the number of any subsequent capturing subpatterns.

The expression `(?:\()` was not captured, but used in follow expression `(?(1)\))`. Here `(1)` was matche to another captured result `(.*?)` and always "true" in this case.

The test fails without the changes.